purchase_company_currency/views/purchase_order_view.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+  <!-- Purchase Order total in company currency -->
+  <record id="purchase_order_tree" model="ir.ui.view">
+    <field name="name">purchase.order.tree</field>
+    <field name="model">purchase.order</field>
+    <field name="inherit_id" ref="purchase.purchase_order_tree" />
+    <field name="arch" type="xml">
+
+      <!-- Remove sums where field is in multiple currencies -->
+      <field name="amount_untaxed" position="attributes">
```

Comment:
@elicoidal These fields are usefull. I'm not removing them. The thing that is been removed is the sum on bottom of it. In your column, you can have amount in euro, dollars,... The sum add them regardless the currency.